transformers/events/DOCUMENTATION.md

```diff
@@ -168,3 +169,9 @@ In the event there are not logs for an event you're looking to transform in ethe
 ## Useful Documents
 
 [Ethereum Event ABI Specification](https://solidity.readthedocs.io/en/develop/abi-spec.html#events)
+
+## Diagram
+
+This shows the entire process, from the perspective of the `execute` process. Writing transformers means writing code for the little loop near the end.
+
+![Transformer Sequence Diagram](./Transforming Events.svg)
```

Comment:
do we need to include this file in this PR?

Reply:
Only if we want it to show up.